outputs.tf

```diff
@@ -23,15 +23,26 @@ output "resource_group_id" {
   description = "The resource group where cloud monitoring monitor instance resides"
 }
 
+output "resource_keys" {
+  description = "Map of resource keys created for the IBM Cloud Monitoring instance, used by agents for authentication and data forwarding."
+  value       = ibm_resource_key.resource_keys
+  sensitive   = true
+}
+
 output "access_key" {
-  value       = ibm_resource_key.resource_key.credentials["Sysdig Access Key"]
+  value       = !var.disable_access_key_creation ? ibm_resource_key.resource_key[0].credentials["Sysdig Access Key"] : null
   description = "The cloud monitoring access key for agents to use"
   sensitive   = true
 }
 
-output "manager_key_name" {
-  value       = ibm_resource_key.resource_key.name
-  description = "The cloud monitoring manager key name"
+# https://cloud.ibm.com/docs/monitoring?topic=monitoring-access_key
+output "access_keys" {
```

Comment:
This is no longer relevant right? We have `resource_keys` and `access_key` outputs already